README.md

```diff
@@ -139,55 +139,25 @@ import utc from '@momentjs/moment/timezones/utc/'; // Note trailing slash
 // Error: folders cannot be imported (there is no index.* magic)
 ```
 
-### 2. Default Field
+### 2. Conditional Mapping
 
-The `default` package.json field can be thought of as the "new main" for Node.js.
+Conditional mapping is an extension of `"exports"` that allows defining different mappings between different environments, for example having a different module path between Node.js and the browser.
 
-Like exports, the target value can be a string or array, must start with './' and be an exact file path - no extension searching applies.
-
-When there is no "default" field, the traditional "main" and its associated lookup will be used, including the default `index.js` main.
-
-"default" can also be set to _false_ to indicate that there is no main entry point for the package.
-
-If any value for "default" set set, the package is immediately encapsulated as if `"exports": {}` were set.
-
-#### Example
-
-```js
-{
-  "main": "./index.js",
-  "default": "./main.js"
-}
-```
-
-will load `pkg/main.js` from both CommonJS and ESM importers `require('pkg')` or `import 'pkg'`.
-
-```js
-{
-  "default": false
-}
-```
-
-will give an error - "No main entry point found for pkg" when trying to `require('pkg')` or `import 'pkg'`.
-
-### 3. Conditional Mapping
-
-Conditional mapping is an extension of the `"exports"` and `"default"` features that allows defining different mappings between different environments, for example having a different module path between Node.js and the browser.
-
-Conditional mappings are defined as _objects_ in the target slot for both `"exports"` and the `"default"` field.
+Conditional mappings are defined as _objects_ in the target slot for `"exports"`.
 
 The object has keys which are _condition names_, and values which correspond to the mapping target.
 
-Condition names are matched in a priority order. In Node.js the following conditions apply in priority order:
+In Node.js, condition names are matched in object order. The first condition name that matches a known condition name for the current environment will match. Node.js supports the following conditions:
 
-1. `"require"`: Indicates we are resolving from a CommonJS importer.
-2. `"node"`: Indicates we are in a Node.js environment.
+1. `"node"`: Indicates we are in a Node.js environment.
+2. `"main"`: Default fallback condition for any generic JS environment.
```

Comment:
Should we call this `module` or `default` instead? So it's not conflated with the traditionally-CommonJS `main` field?

Reply:
Since we're proposing `require` to match CJS then wouldn't `import` make sense to match ES modules?  This way the condition would consistently specify "how it's being loaded" rather than sometimes specifying what is being loaded.

Reply:
The intend was to have a key that doesn't imply anything about either format or how it's being loaded. Just an "else". So maybe "default" is the least confusing.

Reply:
Ahh I saw this and my brain injected "generic ES module JS environment".

Reply:
I've updated this to `"module"`. I would prefer to keep `"default"` free for a new field to avoid introducing a similar confusion!